### PR TITLE
Reorder and add https://lnnodeinsight.com/ to the discover page with a new category: Node management

### LINKED ISSUE #2111: Reorder and add https://lnnodeinsight.com/ to the discover page with a new category: Node management
### Feature description

add channel.ninja, amboss.space and lnnodeinsight to a new category: Node Management

Jakub suggested this order: Entertainment, Shopping, Nostr, Trading, Gaming, Misc.

### Describe the solution

new project and new section on the "Discover" page. 

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

None

### PR DESCRIPTION
**Pull request by SiddheshKukade**
### Describe the changes you have made in this PR
Reordered the sections on the Discover page and added  new section of node management with lnnodeinsight.com as a card.
### Link this PR to an issue [optional]

Fixes   #2111

### Type of change
 

- `refactor`: refactored the sections of the discover page.
- `feat`: New feature (non-breaking change which adds functionality)
 

### Screenshots of the changes [optional]
image
image
### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by SiddheshKukade** [comment]
Please assign someone to review.

File: src/app/screens/Discover/websites.json [hunk 1]
```diff
@@ -162,6 +110,81 @@
...
+      {
+        "title": "Amboss",
+        "subtitle": "Lightning Network node explorer",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/amboss-space_thumbnail.png",
+        "url": "https://amboss.space/"
+      },
+      {
+        "title": "LNnodeinsight.com",
+        "subtitle": "Develop your own data-driven Lightning Network insight",
+        "logo": "https://lnnodeinsight.com/www/LNnodeinsight-tinylogonobkgrnd-white.svg",
```
Comment (reneaaron) [inline src/app/screens/Discover/websites.json:183]: @MoritzKa Can you please upload the logos files to our CDN and replace the paths here?

**Review by reneaaron — COMMENTED** [review]
Thanks for the PR @SiddheshKukade, looks good! 🙌 

Will merge as soon as @MoritzKa replaced the images.

File: src/app/screens/Discover/websites.json (hunk 1 above)
Comment (escapedcat) [reply src/app/screens/Discover/websites.json:183]: Try this:
https://cdn.getalby-assets.com/alby-extension-website-screen/LNnodeinsight-tinylogonobkgrnd.svg

Comment (SiddheshKukade) [reply src/app/screens/Discover/websites.json]: Done, Here's how it looks :
image

**Comment by SiddheshKukade** [comment]
My apologies for the extra commits. I accidentally pushed them into the wrong branch. 
@MoritzKa  Sir, the code is ready to be reviewed.

**Comment by MoritzKa** [comment]
The UI looks good and links are working. I did not check the code though.

**Review by escapedcat — APPROVED** [review]

**Comment by SiddheshKukade** [comment]
> The UI looks good and links are working. I did not check the code though.

@MoritzKa the changes are mostly some JSON refactoring and added code for Node management and added section for LND nodeinsight.com. Let me know if anything's unclear in code.

**Comment by reneaaron** [comment]
Thanks for your work on this PR @SiddheshKukade 🙌

**Comment by SiddheshKukade** [comment]
@reneaaron it's my pleasure ☺️

**Pull request merged**